identity/src/main/java/com/stripe/android/identity/ui/ErrorScreen.kt

```diff
@@ -0,0 +1,133 @@
+package com.stripe.android.identity.ui
+
+import androidx.compose.foundation.Image
+import androidx.compose.foundation.layout.Column
+import androidx.compose.foundation.layout.Spacer
+import androidx.compose.foundation.layout.fillMaxSize
+import androidx.compose.foundation.layout.fillMaxWidth
+import androidx.compose.foundation.layout.height
+import androidx.compose.foundation.layout.padding
+import androidx.compose.foundation.layout.width
+import androidx.compose.foundation.rememberScrollState
+import androidx.compose.foundation.verticalScroll
+import androidx.compose.material.Button
+import androidx.compose.material.OutlinedButton
+import androidx.compose.material.Text
+import androidx.compose.runtime.Composable
+import androidx.compose.ui.Alignment
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.platform.testTag
+import androidx.compose.ui.res.dimensionResource
+import androidx.compose.ui.res.painterResource
+import androidx.compose.ui.res.stringResource
+import androidx.compose.ui.text.font.FontWeight
+import androidx.compose.ui.text.style.TextAlign
+import androidx.compose.ui.unit.dp
+import androidx.compose.ui.unit.sp
+import com.google.android.material.composethemeadapter.MdcTheme
+import com.stripe.android.identity.R
+
+@Composable
+internal fun ErrorScreen(
+    title: String,
+    message1: String? = null,
+    message2: String? = null,
+    topButton: Pair<String, () -> Unit>? = null,
+    bottomButton: Pair<String, () -> Unit>? = null,
```

Comment:
Instead of passing in tuples, can we just have two arguments `topButtonLabel` and `onTopButtonPressed`?

Reply:
sure, but just trying to save some param passed in here, otherwise we would need to null check the two fields of bottomButton, which becomes pretty verbose

Reply:
IMHO  it’s better to hide the button impl details within, as it’s required by design that all the upper button are Outlined Button(Type2 [here](https://m2.material.io/components/buttons#usage)) and bottom button are Contained Button(Type3 [here](https://m2.material.io/components/buttons#usage)), also the test tags of the buttons are defined within the error screen. If they are extracted out then these details would be exposed

Reply:
I can probably create a data class to hold the button details (text, clickLsntr) if that’s easier to read?

Reply:
done!